kedro/framework/cli/starters.py

```diff
@@ -288,7 +399,7 @@ def _get_cookiecutter_dir(
                 f" Specified tag {checkout}. The following tags are available: "
                 + ", ".join(_get_available_tags(template_path))
             )
-        official_starters = sorted(_STARTER_ALIASES)
+        official_starters = sorted(_STARTERS_ALIASES)
```

Comment:
It will return only inbuilt starters: the dictionarry is always deepcopied when ``_get_starters_aliases`` is called, so I think the message is still giving the right result. But maybe it would be better to call explicitly ``_get_starters_aliases()`` here and modify the message accordingly?

Reply:
Ah, sorry I wasn't clear here. The problem I see isn't so much that this shows only inbuilt starters (though maybe that is a small problem also) but what the exception message will say. Won't this line print out the whole dictionary, with all its subkeys etc., rather than just showing the alias names of the starters?
```
yaml.safe_dump(official_starters)
```